include/prereq-build.mk

```diff
@@ -141,10 +141,11 @@ $(eval $(call SetupHostCommand,wget,Please install GNU 'wget', \
 $(eval $(call SetupHostCommand,perl,Please install Perl 5.x, \
 	perl --version | grep "perl.*v5"))
 
-$(eval $(call SetupHostCommand,python,Please install Python 2.x, \
-	python2.7 -V 2>&1 | grep Python, \
-	python2 -V 2>&1 | grep Python, \
-	python -V 2>&1 | grep Python))
+$(eval $(call SetupHostCommand,python,Please install Python 3.x, \
+	python3 -V 2>&1 | grep 'Python 3'))
+
+$(eval $(call SetupHostCommand,python3,Please install Python 3.x, \
```

Comment:
Why two separate entries?

Reply:
To get both `python` and `python3` symlinks pointing to Python 3 binary:

```
staging_dir/host/bin/python -> /usr/bin/python3
staging_dir/host/bin/python3 -> /usr/bin/python3
```

without that I wasn't able to force use of Python 3 on hosts with Python 2 and Python 3 installed, where it's usually Python 2 installed as /usr/bin/python so all build packages searching for Python would use Python 2 version instead of Python 3.